<launch>
  <node name="camera_robot_calibration" pkg="camera_robot_calibration" type="camera_robot_calibration.py" output="screen" >
    <param name="base_frame_name" value="base_link"/>
    <param name="camera_frame_name" value="usb_cam"/>
    <param name="camera_name" value="usb_cam"/>
    <param name="robot_ee_frame_name" value="l_gripper_tool_frame"/>
    <param name="marker_frame_name" value="id_0"/>
  </node>
</launch>
